blog/2022-05-07-release/index.md

```diff
@@ -0,0 +1,257 @@
+
+## L3 cache and MBA resource isolation
+
+Pods of different priorities are usually deployed on the same machine. This may cause tasks to compete for computing
+resources. As a result, the quality of service (QoS) of your service cannot be ensured. The Resource Director
+Technology (RDT) controls the Last Level Cache (L3 cache) that can be used by workloads of different priorities. RDT
+also uses the Memory Bandwidth Allocation (MBA) feature to control the memory bandwidth that can be used by workloads.
+This isolates the L3 cache and memory bandwidth used by workloads, ensures the QoS of high-priority workloads, and
+improves overall resource utilization. This topic describes how to improve the resource isolation of tasks with
+different priorities by controlling the L3 cache and using the MBA feature.
+
+### How to use L3 cache and MBA resource isolation
+
+- Use a ConfigMap to enable L3 cache and MBA resource isolation for all pods in a cluster
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: slo-controller-config
+  namespace: koordinator-system
+data:
+  resource-qos-config: |-
+    {
+      "clusterStrategy": {
+        "lsClass": {
+           "resctrlQoS": {
+             "enable": true,
+             "catRangeStartPercent": 0,
+             "catRangeEndPercent": 100,
+             "MBAPercent": 100
+           }
+         },
+        "beClass": {
+           "resctrlQoS": {
+             "enable": true
+             "catRangeStartPercent": 0,
+             "catRangeEndPercent": 30,
+             "MBAPercent": 100
+           }
+         }
+      }
+    }
+```
+
+## Memory QoS
+
+The Koordlet provides the memory quality of service (QoS) feature for containers. You can use this
+feature to optimize the performance of memory-sensitive applications while ensuring fair memory scheduling among
+containers. This topic describes how to enable the memory QoS feature for containers.
+
+### Background information
+
+The following memory limits apply to containers:
+
+- The memory limit of the container. If the amount of memory that a container uses, including the page cache, is about
+  to reach the memory limit of the container, the memory reclaim mechanism of the OS kernel is triggered. As a result,
+  the application in the container may not be able to request or release memory resources as normal.
+- The memory limit of the node. If the memory limit of a container is greater than the memory request of the container,
+  the container can overcommit memory resources. In this case, the available memory on the node may become insufficient.
+  This causes the OS kernel to reclaim memory from containers. As a result, the performance of your application is
+  downgraded. In extreme cases, the node cannot run as normal.
+
+To improve the performance of applications and the stability of nodes, Koordinator provides the memory QoS feature for
+containers. **To use this feature, you must use Anolis OS as the node OS.** After
```

Comment:
Recommended, not mandatory. On other OS, we will try our best to adapt, and users can enable it without side effects.